Makefile

```diff
@@ -85,10 +92,11 @@ logs: ## display grafana logs (follow mode)
 	@$(COMPOSE) logs -f grafana
 .PHONY: logs
 
-plugins: \
-	tree \
-	var/lib/grafana/plugins/grafana-groupedbarchart-panel-master
-plugins: ## download plugins
+plugins: ## download, build and install plugins
+	for plugin in $(source_plugins) ; do \
+		$(COMPOSE_RUN) node yarn --cwd=$$plugin --no-default-rc --use-yarnrc="./.yarnrc" install; \
```

Comment:
:thinking: not sure you need it as the plugin has already been installed in `dependencies`.

Reply:
Good point) Thanks) Indeed, some dependencies are installed twice in this case. Updated this part with `yarn workspaces` and the `lerna` package to reduce duplicated package installations.